drivers/ethernet/phy/Kconfig.dm8806

```
@@ -29,6 +29,14 @@ config PHY_DM8806_TRIGGER_GLOBAL_THREAD
 
 endchoice
 
+config PHY_DM8806_ENABLE_ENERGY_EFFICIENT_MODE
+	bool "802.3az Energy efficient mode"
+	default n
```

Omit `default n`, and I wouldn't put `ENABLE` in a boolean option.

```suggestion
config PHY_DM8806_ENERGY_EFFICIENT_MODE
	bool "802.3az Energy efficient mode"
```

---

I will take into account this remark. But because of conflicts with different fork (https://github.com/zephyrproject-rtos/zephyr/pull/84298), I would like to avoid a mess and I will remove this PR and create another one with your suggestion. Unless there is another way to resolve this conflict?

---

Where is the conflict? Simply modify your branch of this PR locally and force push?